src/dataDisplay/list/list.jsx

```diff
@@ -40,23 +40,23 @@ const propTypes = {
      * The content of the subheader, normally `ListSubheader`.
      */
     subheader: PropTypes.node,
-    version: PropTypes.number,
+    designVersion: PropTypes.number,
```

Comment:
Same NAB here =)

Reply:
Updated.

Reply:
Looks good.  Resolving.
(Note: this is now in `src/dataDisplay/list/listBase.jsx`)